### ISSUE TITLE
Clarify the role of non-coding contributors and maintainers

### ISSUE THREAD

**Issue by adamhjk** [adamhjk/community-compact#3]
We should clarify that you can be a contributor and a maintainer without writing code.

**Comment by thommay** [comment]
If this does get clarified it would be delightful to flip the whole thing and clarify that only some contributors/maintainers contribute code (and hence doing so is not a pre-req).

**Comment by adamhjk** [comment]
Yes. Do we think the right word is "maintainer"? Lots of different ways to maintain the communities health. Contributor I think implies software for a lot of people.

**Comment by eeyun** [comment]
I go back and forth on that verbiage myself. It's a tough one. I think you're likely correct that there is an implication with the phrasing of Contributor that said contribution is typically code. I think what I come back to most of the time with these two phrases is more frequently around the level of engagement for the individual. 

Basically, the way I've historically thought about it is all maintainers are inherently some kind of contributor but not all contributors are inherently maintainers and the difference between is that one is likely engaged at a deeper level frequently over a longer period of time.

Maybe this isn't a helpful thought :joy: ?
